setup.cfg

```diff
@@ -143,6 +143,7 @@ docs =
     napari-sphinx-theme
     sphinx-external-toc
     lxml
+    qtgallery
```

Comment:
do we know how the gallery's worked so far without this import? Were we importing it elsewhere?

Reply:
this is not import. And build-docs locally fail because of lack of qtgallery. See that this is `docs` extra